libraries/chain/snapshot_scheduler.cpp

```diff
@@ -203,13 +211,7 @@ void snapshot_scheduler::create_snapshot(next_function<snapshot_information> nex
    auto& pending_by_id = _pending_snapshot_index.get<by_id>();
    auto existing = pending_by_id.find(head_id);
    if(existing != pending_by_id.end()) {
```

Comment:
Why not change the if condition to `if(existing == pending_by_id.end()) {` to avoid an empty `if` clause?

Reply:
https://github.com/AntelopeIO/spring/pull/1538/commits/bf0c467da6f5294340cdf8ffd4323aa3f3d29f3d